galley/pkg/testing/testdata/dataset/extensions/v1beta1/ingress_merge_0_expected.json

```diff
@@ -74,7 +74,7 @@
             "route": [
               {
                 "destination": {
-                  "host": "service2.ns.svc.cluster.local",
+                  "host": "service2.{{.Namespace}}.svc.cluster.local",
```

Comment:
the second part (`.svc.cluster.local`) is configured as the `Domain`.  Not sure if this is something you can add here as well.

Reply:
I want to focus on Galley tests and converting them into compliance suite later. I think that something we must do, but I'd rather do a more focused PR for that purpsoe.